<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCCleaner</class>
 <widget class="QMainWindow" name="DCCleaner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCCleaner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>로그인</string>
    </property>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="isSaveAccount">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>로그인 정보 저장</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ID</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>PW</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="loginStatus">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인 상태 : None</string>
     </property>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="devInfoButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>301</width>
      <height>48</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string> Made by qwertycvb(SerenityS)</string>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>게시글 클리너</string>
    </property>
    <widget class="QComboBox" name="postGallList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="totalPost">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>전체 게시글 :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delPost">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>게시글 삭제</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>댓글 클리너</string>
    </property>
    <widget class="QComboBox" name="commentGallList">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="totalComment">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>전체 댓글 :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delComment">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>댓글 삭제</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
